docs/_template.html

```diff
@@ -103,8 +103,26 @@
   else{
     document.getElementById(e.target.id+'-wrapper').classList.add('menu-open');
   }
-})
-});
+      })
+      let fsdocsFileName = `{{fsdocs-source-filename}}`
+      let fsdocsSection = fsdocsFileName.substring(fsdocsFileName.indexOf('/') + 1, fsdocsFileName.lastIndexOf('/'))
```

Comment:
This probably won't work for API doc pages like https://github.com/fsprojects/fantomas/pull/2473/files